src/api/smt2tokens.h

```diff
@@ -26,51 +26,96 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 #ifndef APITOKENS_H
 #define APITOKENS_H
+
+#include <string>
+#include <unordered_map>
+namespace osmttokens {
 //
 // The names for the tokens in the API for smtlib
 //
-enum token {
-    t_none,
-    t_as,
-    t_DECIMAL,
-    t_NUMERAL,
-    t_par,
-    t_STRING,
-    t_exists,
-    t_forall,
-    t_assert,
-    t_checksat,
-    t_declaresort,
-    t_definesort,
-    t_declarefun,
-    t_declareconst,
-    t_definefun,
-    t_exit,
-    t_getassertions,
-    t_getassignment,
-    t_getinfo,
-    t_setinfo,
-    t_getoption,
-    t_setoption,
-    t_getproof,
-    t_getunsatcore,
-    t_getvalue,
-    t_getmodel,
-    t_pop,
-    t_push,
-    t_setlogic,
-    t_getinterpolants,
-    t_theory,
-    t_writestate,
-    t_readstate,
-    t_simplify,
-    t_writefuns,
-    t_let,
-    t_echo
-};
+    enum token {
+        t_none,
+        t_as,
+        t_DECIMAL,
+        t_NUMERAL,
+        t_par,
+        t_STRING,
+        t_exists,
+        t_forall,
+        t_assert,
+        t_checksat,
+        t_declaresort,
+        t_definesort,
+        t_declarefun,
+        t_declareconst,
+        t_definefun,
+        t_exit,
+        t_getassertions,
+        t_getassignment,
+        t_getinfo,
+        t_setinfo,
+        t_getoption,
+        t_setoption,
+        t_getproof,
+        t_getunsatcore,
+        t_getvalue,
+        t_getmodel,
+        t_pop,
+        t_push,
+        t_setlogic,
+        t_getinterpolants,
+        t_theory,
+        t_writestate,
+        t_readstate,
+        t_simplify,
+        t_writefuns,
+        t_let,
+        t_echo
+    };
+
+    inline const std::unordered_map<token, std::string> tokenToName = {
```

Comment:
`constexpr` should also work here, no?
```suggestion
    inline constexpr std::unordered_map<token, std::string> tokenToName = {
```

Reply:
It seems `unordered_map` cannot be `constexpr` yet.